src/algebra/continued_fractions/computation/approximations.lean

```diff
@@ -3,17 +3,24 @@ Copyright (c) 2020 Kevin Kappelmann. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kevin Kappelmann
 -/
-import algebra.continued_fractions.computation.translations
-import algebra.continued_fractions.continuants_recurrence
-import algebra.continued_fractions.terminated_stable
+import algebra.continued_fractions.computation.correctness_terminating
 import data.nat.fib
+import tactic.solve_by_elim
 /-!
 # Approximations for Continued Fraction Computations (`gcf.of`)
 
 ## Summary
 
 Let us write `gcf` for `generalized_continued_fraction`. This file contains useful approximations
 for the values involved in the continued fractions computation `gcf.of`.
+In particular, we derive the so-called *determinant formula* for `gcf.of`:
+`Aₙ * Bₙ₊₁ - Bₙ * Aₙ₊₁ = (-1)^(n + 1)`.
+
+Moreover, we derive some upper bounds for the error term when computing a continued fraction up a
+given position, i.e. bounds for the term `|v - (gcf.of v).convergents n|`.
+The derived bounds will show us that the error term indeed gets smaller.
+As a corollary, we will be able to show that `(gcf.of v).convergents` converges to `v`.
+(TODO add reference to convergence lemma in next PR)
```

Comment:
We usually have a section
```
## TODO
```
below `Main Theorems`. That might make it a bit easier to find.